packages/next/src/server/lib/incremental-cache/file-system-cache.ts

```diff
@@ -71,7 +71,7 @@ export default class FileSystemCache implements CacheHandler {
   }
 
   private loadTagsManifest() {
-    if (!this.tagsManifestPath || !this.fs) return
+    if (!this.tagsManifestPath || !this.fs || tagsManifest) return
```

Comment:
can we replace this fn by the loadManifest fn I added? essentially does the same thing I think